File: 00-Code/Android/UISystem/View/src/main/java/com/ztiany/view/ContentActivity.java
package com.ztiany.view;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.util.Log;
import android.view.View;

/**
 * @author Ztiany
 *         Email: ztiany3@gmail.com
 *         Date : 2017-08-05 15:27
 */
public class ContentActivity extends AppCompatActivity {

    private Toolbar mToolbar;

    public static Intent getLaunchIntent(Context context, String title, Class fragment) {
        Intent intent = new Intent(context, ContentActivity.class);
        intent.putExtra("key1", title);
        intent.putExtra("key2", fragment.getName());
        return intent;
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.common_activity_content);

        mToolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(mToolbar);
        mToolbar.setContentInsetStartWithNavigation(0);
        if (getSupportActionBar() != null) {
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
            mToolbar.setNavigationOnClickListener(new View.OnClickListener() {
                @Override
                public void onClick(View v) {
                    supportFinishAfterTransition();
                }
            });
        }

        if (savedInstanceState == null) {
            String name = getIntent().getStringExtra("key2");
            getSupportFragmentManager()
                    .beginTransaction()
                    .replace(R.id.fl_content, Fragment.instantiate(this, name, null), name)
                    .commit();
        }
    }

    @Override
    protected void onResume() {
        super.onResume();
        String title = getIntent().getStringExtra("key1");
        Log.d("ContentActivity", title);
        mToolbar.setTitle(title);
    }
}

File: 00-Code/Android/AAC-Sample/Paging-Sample/src/main/java/com/ztiany/acc/ContentActivity.kt
package com.ztiany.acc

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import com.ztiany.acc.live.LiveDataFragment
import com.ztiany.acc.rx.RxJavaFragment

class ContentActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_content)
        val type = intent.getStringExtra(MainActivity.TYPE)
        if (type == MainActivity.LIVE_DATA) {
            showFragment(LiveDataFragment())
        } else if (type == MainActivity.RXJAVA) {
            showFragment(RxJavaFragment())
        }
    }

    private fun showFragment(fragment: Fragment) {
        supportFragmentManager
                .beginTransaction()
                .add(R.id.fl_content, fragment)
                .commit()
    }

}
